.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    # Using mypyc-compiled black 👆 (faster)
+    rev: 23.7.0
+    hooks:
+      - id: black-jupyter
+        # The latest version of Python supported by the project.
+        language_version: python3.10
```

Comment:
- [ ] Check this with Matt. [dxh maxes at 3.10](https://github.com/UCL/dxh/blob/main/pyproject.toml#L17).